src/server/game/Handlers/CharacterHandler.cpp

```diff
@@ -341,6 +341,13 @@ void WorldSession::HandleCharCreateOpcode(WorldPacket& recvData)
 
     if (!HasPermission(rbac::RBAC_PERM_SKIP_CHECK_CHARACTER_CREATION_RACEMASK))
     {
+        if (raceEntry->Alliance == ChrRacesAllianceType::CHRRACES_ALLIANCE_TYPE_NOT_PLAYABLE || raceEntry->HasFlag(ChrRacesFlags::CHRRACES_FLAGS_NOT_PLAYABLE))
```

Comment:
Codestyle remark: dont prefix regular enums with `Type::`, use that only for `enum class` where its required

Reply:
Ok, I will change this but I'm curious if I should just use enum class whenever possible? Always seems so very strange coming from C#.